supplementary_style_guide/glossary_terms_conventions/general_conventions/f.adoc

```diff
@@ -247,7 +247,7 @@ Do not use "Firewire" or "firewire". Although FireWire is a trademark of Apple C
 [discrete]
 [[fuzzy]]
 ==== image:images/caution.png[with caution] fuzzy (adjective)
-*Description*: It is only correct to use "fuzzy" as an adjective when referring to "fuzzy searches" (the technique of finding strings that match a pattern approximately, rather than exactly). See http://www.stylepedia.net/#chap-Red_Hat_Technical_Publications-Writing_Style_Guide-Avoiding_Slang_Metaphors_and_Misleading_Language[Avoiding Slang, Metaphors, and Misleading Language] for details and examples.
+*Description*: It is only correct to use _fuzzy_ as an adjective when referring to "fuzzy searches" (the technique of finding strings that match a pattern approximately, rather than exactly). See http://www.stylepedia.net/#chap-Red_Hat_Technical_Publications-Writing_Style_Guide-Avoiding_Slang_Metaphors_and_Misleading_Language[Avoiding Slang, Metaphors, and Misleading Language] for details and examples.
```

Comment:
This should stay quotes since it's telling how it should be written

Reply:
:heavy_check_mark: